client/fresheyes-web/src/app/components/form/FormSection.tsx

```diff
@@ -1,18 +1,19 @@
 "use client";
 
 import React, { useState } from "react";
-import { PullRequest } from "@/proto/fresheyes_pb";
 
 import Modal from "../modal/Modal";
-import { useGrpcClient } from "@/app/hooks/useGrpcClinet";
+import { processPr } from "./proceess-pr";
 
 const FormSection = () => {
-  const { client } = useGrpcClient();
-
   const [link, setLink] = useState("");
   const [loading, setLoading] = useState({ loader: false, modal: false });
   const [error, setError] = useState("");
-  const [formValues, setFormValues] = useState<{ owner: string; repo: string; pull_number: number }>({
+  const [formValues, setFormValues] = useState<{
```

Comment:
I think we can use the `PullRequest` type definition here

Reply:
addressed here -> https://github.com/Extheoisah/fresheyes-ref-impl/pull/11/commits/e2eb89e3735ae3fce11c967b4dcb9279f6762479